website/content/docs/configuration/storage/postgresql.mdx

```diff
@@ -40,8 +40,68 @@ to disable SSL.
 
 :::
 
-The PostgreSQL storage backend does not automatically create the table. Here is
-some sample SQL to create the schema and indexes.
+## `postgresql` parameters
+
+- `connection_url` `(string: <required>)` – Specifies the connection string to
+  use to authenticate and connect to PostgreSQL. The connection URL can also be
+  set using the `BAO_PG_CONNECTION_URL` environment variable. A full list of supported
+  parameters can be found in the [pgx library][pgxlib] and [PostgreSQL connection string][pg_conn_docs]
+  documentation. For example connection string URLs, see the examples section below.
+
+- `table` `(string: "openbao_kv_store")` – Specifies the name of the table in
+  which to write OpenBao data. This table must already exist (OpenBao will not
+  attempt to create it).
```

Comment:
I'm a little but confused: some tables of the DB will be automatically created by OpenBao but not all of them ?
In this case, we should probably mention the tables that are automatically created ?
Furthermore, if the user enables the KV engine, maybe we should automatically add the table for this ?

Reply:
@DanGhita Pushed :-)